.github/actions/update-production-dependencies-list/action.yml

```diff
@@ -0,0 +1,9 @@
+name: 'Update PRODUCTION_DEPENDENCIES.md'
+description: 'Update list in PRODUCTION_DEPENDENCIES.md'
+inputs:
+  dependencies-from-license-checker:
+    description: 'Output from license-checker'
+    required: true
+runs:
+  using: 'node16'
```

Comment:
question: can we use the `node-version-file` as we do in other pipelines?

Reply:
Have changed the action to be `composite` like the other actions now. For javascript actions you can't specify any node-version-file.